isis/src/base/objs/CameraPointInfo/CameraPointInfo.cpp

```diff
@@ -405,17 +403,27 @@ namespace Isis {
 
       double pB[3], spB[3], sB[3];
       QString utc;
-      double ssplat, ssplon, sslat, sslon, ocentricLat, ographicLat, pe360Lon, pw360Lon;
+      double ssplat, ssplon, ocentricLat, ographicLat, pe360Lon, pw360Lon;
 
       {
         gp->findKeyword("FileName").setValue(m_currentCube->fileName());
         gp->findKeyword("Sample").setValue(toString(m_camera->Sample()));
         gp->findKeyword("Line").setValue(toString(m_camera->Line()));
         gp->findKeyword("PixelValue").setValue(PixelToString(b[0]));
-        gp->findKeyword("RightAscension").setValue(toString(
-                        m_camera->RightAscension()), "DEGREE");
-        gp->findKeyword("Declination").setValue(toString(
-                        m_camera->Declination()), "DEGREE");
+        try {
+          gp->findKeyword("RightAscension").setValue(toString(
+                          m_camera->RightAscension()), "DEGREE");
+        }
+        catch (IException &e) {
+          gp->findKeyword("RightAscension").setValue("Null");
```

Comment:
This is tricky because PvlKeywords default to no value at all. I.E.

```
Group = SomeGroup
  SomeKeyword
EndGroup
```

There is also logic where values are iteratively appended to the keyword and that would be tricky if the default value is null, not empty.

I agree this is clunky. Maybe a simple function to set it to null? so these would be like

```
gp->findKeyword("RightAscension").nullify()
```

Reply:
Talked about this over teams. This is fine for the time being and we often don't deal with null keyword values in ISIS rather, we check if the keyword is present. Seeing as we are trying to fit CSM into ISIS, this current system of setting these keywords to null manually is okay